src/styles/global.scss

```diff
@@ -14,6 +14,81 @@
   }
 }
 
+html.is-extension {
+  width: $extension-width;
+  height: $extension-height;
+
+  // fixes issue with extension not rendering properly
+  ion-tabs,
+  .tabs-inner,
+  ion-router-outlet,
+  .ion-page {
+    contain: layout style !important;
```

Comment:
Why do we need to use `!important` here? We are avoiding usage of it.

Reply:
This overrides some css that is imported by Ionic that is causing issues to the extension on FF